Reachable unwrap in sys_rt_sigprocmask()

### Describe the bug
There is a reachable unwrap in `sys_rt_sigprocmask()` at kernel/aster-nix/src/syscall/rt_sigprocmask.rs:21 when make a `rt_sigprocmask ` syscall with specific arguments.

https://github.com/asterinas/asterinas/blob/4844e7ca7ca6d78896a51a71487a6fdfe9ca6654/kernel/aster-nix/src/syscall/rt_sigprocmask.rs#L14-L21

### To Reproduce
1. Compile a program which calls `mmap` with specific arguments:
```C
#include <stdio.h>
#include <sys/syscall.h>
#include <unistd.h>
#include <signal.h>
#include <string.h>
#include <errno.h>

int main() {
    int how = 0xfffc;
    sigset_t set;
    sigset_t oset;
    size_t sigsetsize = 128;

    memset(&set, 0, sizeof(set));
    memset(&oset, 0, sizeof(oset));
    sigaddset(&set, 3);

    long result = syscall(SYS_rt_sigprocmask, how, &set, &oset, sigsetsize);

    return 0;
}
```
2. Run the compiled program in Asterinas.

### Expected behavior

Asterinas reports panic and is terminated.

### Environment

- Official docker asterinas/asterinas:0.6.2
- 11th Gen Intel(R) Core(TM) i7-1165G7 @ 2.80GHz
- Asterinas version: main 4844e7c

### Logs
```
~ # /root/rt_sigprocmask-test 
panicked at /root/asterinas/kernel/aster-nix/src/syscall/rt_sigprocmask.rs:21:41:
called `Result::unwrap()` on an `Err` value: InvalidValue
Printing stack trace:
   1: fn 0xffffffff88838040 - pc 0xffffffff88838058 / registers:

     rax               0x12; rdx 0xffffffff889ede68; rcx                0x1; rbx                0x0;
     rsi                0x0; rdi                0x0; rbp                0x0; rsp 0xffff80007f72a540;

   2: fn 0xffffffff88837e20 - pc 0xffffffff88837f98 / registers:

     rax               0x12; rdx 0xffffffff889ede68; rcx                0x1; rbx                0x0;
     rsi                0x0; rdi                0x0; rbp                0x0; rsp 0xffff80007f72a550;

   3: fn 0xffffffff88049000 - pc 0xffffffff8804900a / registers:

     rax               0x12; rdx 0xffffffff889ede68; rcx                0x1; rbx                0x0;
     rsi                0x0; rdi                0x0; rbp                0x0; rsp 0xffff80007f72a6d0;

   4: fn 0xffffffff8895cec0 - pc 0xffffffff8895cf42 / registers:

     rax               0x12; rdx 0xffffffff889ede68; rcx                0x1; rbx                0x0;
     rsi                0x0; rdi                0x0; rbp                0x0; rsp 0xffff80007f72a6e0;

   5: fn 0xffffffff8893a1f0 - pc 0xffffffff8893a31b / registers:

     rax               0x12; rdx 0xffffffff889ede68; rcx                0x1; rbx                0x0;
     rsi                0x0; rdi                0x0; rbp                0x0; rsp 0xffff80007f72a770;

   6: fn 0xffffffff883c4690 - pc 0xffffffff883c471b / registers:

     rax               0x12; rdx 0xffffffff889ede68; rcx                0x1; rbx                0x0;
     rsi                0x0; rdi                0x0; rbp                0x0; rsp 0xffff80007f72a870;

   7: fn 0xffffffff8840ade0 - pc 0xffffffff88411190 / registers:

     rax               0x12; rdx 0xffffffff889ede68; rcx                0x1; rbx                0x0;
     rsi                0x0; rdi                0x0; rbp                0x0; rsp 0xffff80007f72ab40;

   8: fn 0xffffffff8840ab00 - pc 0xffffffff8840ab8e / registers:

     rax               0x12; rdx 0xffffffff889ede68; rcx                0x1; rbx                0x0;
     rsi                0x0; rdi                0x0; rbp                0x0; rsp 0xffff80007f7403d0;

   9: fn 0xffffffff88524040 - pc 0xffffffff88524bbf / registers:

     rax               0x12; rdx 0xffffffff889ede68; rcx                0x1; rbx                0x0;
     rsi                0x0; rdi                0x0; rbp                0x0; rsp 0xffff80007f740570;

  10: fn 0xffffffff882259d0 - pc 0xffffffff882259de / registers:

     rax               0x12; rdx 0xffffffff889ede68; rcx                0x1; rbx                0x0;
     rsi                0x0; rdi                0x0; rbp                0x0; rsp 0xffff80007f740f90;

  11: fn 0xffffffff8882cbf0 - pc 0xffffffff8882cc06 / registers:

     rax               0x12; rdx 0xffffffff889ede68; rcx                0x1; rbx                0x0;
     rsi                0x0; rdi                0x0; rbp                0x0; rsp 0xffff80007f740fb0;

  12: fn 0xffffffff88798b20 - pc 0xffffffff88798b89 / registers:

     rax               0x12; rdx 0xffffffff889ede68; rcx                0x1; rbx                0x0;
     rsi                0x0; rdi                0x0; rbp                0x0; rsp 0xffff80007f740fd0;

  13: fn                0x0 - pc                0x0 / registers:

     rax               0x12; rdx 0xffffffff889ede68; rcx                0x1; rbx                0x0;
     rsi                0x0; rdi                0x0; rbp                0x0; rsp 0xffff80007f741000;

[OSDK] The kernel seems panicked. Parsing stack trace for source lines:
(  1) /root/asterinas/ostd/src/panicking.rs:107
(  2) /root/asterinas/ostd/src/panicking.rs:59
(  3) au2ndt6r5ksv2yyhocg58xyv7:?
(  4) ??:?
(  5) ??:?
(  6) /root/.rustup/toolchains/nightly-2024-06-20-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/result.rs:1102
(  7) /root/asterinas/kernel/aster-nix/src/syscall/mod.rs:173
(  8) /root/asterinas/kernel/aster-nix/src/syscall/mod.rs:323
(  9) /root/asterinas/kernel/aster-nix/src/thread/task.rs:69
( 10) /root/.rustup/toolchains/nightly-2024-06-20-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ops/function.rs:79
( 11) /root/.rustup/toolchains/nightly-2024-06-20-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/boxed.rs:2077
( 12) /root/asterinas/ostd/src/task/task.rs:295
make: *** [Makefile:153: run] Error 1
```

--- a/kernel/aster-nix/src/syscall/rt_sigprocmask.rs
+++ b/kernel/aster-nix/src/syscall/rt_sigprocmask.rs
@@ -18,7 +18,7 @@ pub fn sys_rt_sigprocmask(
     sigset_size: usize,
     ctx: &Context,
 ) -> Result<SyscallReturn> {
-    let mask_op = MaskOp::try_from(how).unwrap();
+    let mask_op = MaskOp::try_from(how)?;
     debug!(
         "mask op = {:?}, set_ptr = 0x{:x}, oldset_ptr = 0x{:x}, sigset_size = {}",
         mask_op, set_ptr, oldset_ptr, sigset_size

